R/functions_effects_REM.R

```diff
@@ -181,7 +181,7 @@ init_REM_choice.indeg <- function(effectFun, network, window, n1, n2) {
 #' @noRd
 #'
 #' @examples
-#' \dontrun{
+#' \donttest{
```

Comment:
There are a bunch of conditional expressions in this script that codefactor is suggesting we use double `&&` for so that condition-matching occurs stepwise

Reply:
Done in [5fcd81c](https://github.com/snlab-ch/goldfish/pull/77/commits/5fcd81cff3c44c9fbf950afc14b548ae1defb66c)